lumberdev/sum-it-up
# feat: input component

## WHAT IT DOES:

_Description of the pull request, what changed_

- Added three different input interfaces for Website, Text and Song
- Clicking on summarize will make an API request (API endpoint TBD), and will store the response JSON object in localstorage with the key `summaries`
- Added header, footer and About section

## HOW TO TEST:

_Description of how to test if applicable_
Enter a url and click _Summarize_, you should be able to see the response in the console. You could also open the browser Local Storage and  check the value for the key summaries.

### PREVIEW:

_Attach any relevant videos or images:_
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/52301822/215509785-576083c1-01b3-4041-9153-af7a48c4652c.png">


---

### DESIGN:

_Attach link to figma file or design files here:_
[Figma Link](https://www.figma.com/file/JW0F2NoS3I5aqj82C2Lwq0/Sum-It-Up-(Copy)?node-id=68%3A152&t=NnRYlt5KJr1n9h0M-4)
---

### TASK:

### WHAT'S LEFT:

- Update API Endpoints
- Update slider design
- Error handling for the input fields
- Add error page and loading state
- Styling header and footer section
